src/index.js

```diff
@@ -1,14 +1,39 @@
-import _ from 'lodash';
 import './style.css';
 
- function component() {
-   const element = document.createElement('div');
+const arr = [
+  {
+    description: 'Study JS',
+    completed: 'false',
+    index: 1,
+  },
+  {
+    description: 'Study HTML',
+    completed: 'false',
+    index: 2,
+  },
+  {
+    description: 'Study CSS',
+    completed: 'false',
+    index: 3,
+  },
+];
```

Comment:
- Kindly as the previous reviewer request, change the value of the key completed, to `false`. Maybe this makes you confused because you are writing `'false'`, that's a string with the value `'false'`, not a boolean `false`